README.md

```diff
@@ -94,9 +94,9 @@ from griptape.tools import WebScraper, FileManager, TaskMemoryClient
 agent = Agent(
     input_template="Load {{ args[0] }}, summarize it, and store it in a file called {{ args[1] }}.",
     tools=[
-        WebScraper(),
-        FileManager(),
-        TaskMemoryClient(off_prompt=True)
+        WebScraper(off_prompt=True),
+        TaskMemoryClient(off_prompt=True),
+        FileManager()
```

Comment:
FileManager need `off_prompt=False` here to be 1:1 with the original.

Reply:
Do you mean `off_prompt=True`? `FileManager()` is now identical to `FileManager(off_prompt=False)`

Though technically `FileManager` returns an `InfoArtifact` which is not stored in Task Memory, so it doesn't matter. I figured omitting the parameter was slightly better than including one that didn't do anything.

Reply:
Got it, I must've misread something in my original pass.